Cargo.toml

```diff
@@ -30,7 +30,7 @@ parking_lot = "0.12"
 paste = "1.0"
 profiling = "1.0"
 raw-window-handle = "0.5"
-spirq = "=0.6.2"
+spirq = "1.0.0"
```

Comment:
This version encounters a few panics when running `contrib/rel-mgmt/run-all-examples` but https://github.com/PENGUINLIONG/spirq-rs/pull/114 addresses the issues.